draft-ietf-oauth-identity-chaining.md

```diff
@@ -229,6 +241,10 @@ The authorization server MUST validate the JWT authorization grant as specified
 
 The authorization server responds with an access token as described in section 5.1 of {{RFC6749}}.
 
+The following process rule applies:
+
+* If the JWT authorization grant in the access token request contains a "requested_cnf" claim, the authorization server MUST include the "cnf" claim from the "requested_cnf" claim in the returned token.
```

Comment:
If AS of domain B does not support sender constraining tokens it should still be able to issue an access token (but without 'cnf'). Otherwise, chaining from sender constraint domain to non-constraint domain won't be possible.